docs/docs/labs/components/index.md

```diff
@@ -1,7 +1,14 @@
 ---
 title: "Components"
-sidebar_position: 10
+sidebar_class_name: hidden
 ---
+
+:::info
+
+Please contact Dagster if you would like to try Components.
```

Comment:
This is placeholder language -- who should we tell users to contact specifically? cc @yuhan